src/sparseml/yolov8/train.py

```diff
@@ -212,18 +211,15 @@
     "--copy-paste", type=float, default=0.0, help="segment copy-paste (probability)"
 )
 @click.option(
-    "--datasets-dir",
+    "--dataset_path",
```

Comment:
Let's use --dataset-path, making it consistent w/ other arguments (dash instead of underscore)